app_dart/lib/src/service/luci_build_service.dart

```diff
@@ -62,6 +62,12 @@ class LuciBuildService {
   static const int kDefaultPriority = 30;
   static const int kRerunPriority = 29;
 
+  /// Github labels have a max length of 100, so conserve chars here.
+  /// This is currently used by packages repo only.
+  /// See: https://github.com/flutter/flutter/issues/130076
+  static const String githubBuildLabelPrefix = 'gh_override:';
```

Comment:
Why do we need a `gh_` prefix on GitHub labels? What is `gh_` distinguishing from in the context of GitHub label names?

Reply:
If I recall yusuf is porting a functionality that existed in cirrus? What was this defined as previously? Or was it?

Reply:
Cirrus puts all PR labels in an environment variable called `CIRRUS_PR_LABELS`. There is no magic prefix in the Cirrus version of the functionality.